src/modules/design/property-panel/indextabs/general/sections/data-objects/input-association/input-association.ts

```diff
@@ -0,0 +1,32 @@
+import {inject} from 'aurelia-framework';
+
+import {IShape} from '@process-engine/bpmn-elements_contracts';
+
+import {HelpTextId, ISection} from '../../../../../../../../contracts/index';
+import {HelpModalService} from '../../../../../../../../services/help-modal-service/help-modal-service';
+
+@inject(HelpModalService)
+export class InputAssociationSection implements ISection {
+  public path: string = '/sections/data-objects/input-association/input-association';
```

Comment:
Die Associations nennen sich "DataInputAssociation", bzw. "DataOutputAssociation". Sollte man auch hier bei der internen Verwendung entsprechend ausschreiben.

```suggestion
  public path: string = '/sections/data-objects/input-association/data-input-association';
```